### PR TITLE
[BACKLOG-11918] MetaInject and ElasticSearch help documentation update

### PR DESCRIPTION
**Pull request by davidmsantos90**
@mdamour1976 please review

### CODE REVIEW

File: engine/src/org/pentaho/di/trans/step/messages/messages_en_US.properties [hunk 1]
```diff
@@ -519,3 +519,7 @@ BaseStep.Log.ReleasedServerSocketOnPort = Released server socket on port {0}
 BaseStep.Log.RunningOnSlaveServer = Running on slave server #{0}/{1}.
 BaseStep.Log.OpenedServerSocketConnectionTo = Opened a server socket connection to {0}
 BaseStep.Log.SelectedRemoteOutputStepToServer = Selected remote output step [{0}] to open a server socket to remote step [{1}].{2} on port {3}
+
+# Documentation Urls
+BaseStep.DocumentationUrl.MetaInject=https://help.pentaho.com/Documentation/7.0/0L0/0Y0/0K0/ETL_Metadata_Injection
```
Comment (mdamour1976) [inline engine/src/org/pentaho/di/trans/step/messages/messages_en_US.properties]: Let's make use of Const.getBaseDocUrl and only use a fragment here:  0L0/0Y0/0K0/ETL_Metadata_Injection

Also, let's not use "0L0/0Y0/0K0/ETL_Metadata_Injection" here anyway.  This really needs to go in the step itself, via annotation / documentationUrl (and using a fragment in that case will also work, as plugins are processed in a way that will automatically make use of the baseDocUrl.

File: plugins/elasticsearch-bulk-insert/src/org/pentaho/di/trans/steps/elasticsearchbulk/ElasticSearchBulkMeta.java [hunk 2]
```diff
@@ -66,7 +66,7 @@
     name = "ElasticSearchBulk.TypeLongDesc.ElasticSearchBulk",
     description = "ElasticSearchBulk.TypeTooltipDesc.ElasticSearchBulk",
     categoryDescription = "i18n:org.pentaho.di.trans.step:BaseStep.Category.Bulk", image = "ESB.svg",
-    documentationUrl = "http://wiki.pentaho.com/display/EAI/ElasticSearch+Bulk+Insert" )
+    documentationUrl = "i18n:org.pentaho.di.trans.step:BaseStep.DocumentationUrl.ElasticSearchBulk" )
```
Comment (mdamour1976) [inline plugins/elasticsearch-bulk-insert/src/org/pentaho/di/trans/steps/elasticsearchbulk/ElasticSearchBulkMeta.java]: Technically all we need here is "0L0/0Y0/0K0/ElasticSearch_Bulk_Insert"

File: plugins/meta-inject-plugin/src/main/java/org/pentaho/di/trans/steps/metainject/MetaInjectMeta.java [hunk 3]
```diff
@@ -73,8 +73,11 @@
  * @version 3.0
  * @since 2007-07-05
  */
-@Step( id = "MetaInject", image = "org/pentaho/di/ui/trans/steps/metainject/img/GenericTransform.svg", name = "ETL "
-      + "Metadata Injection", categoryDescription = "i18n:org.pentaho.di.trans.step:BaseStep.Category.Flow" )
+@Step( id = "MetaInject",
+    image = "org/pentaho/di/ui/trans/steps/metainject/img/GenericTransform.svg",
+    name = "i18n:org.pentaho.di.trans.step:BaseStep.TypeLongDesc.MetaInject",
+    categoryDescription = "i18n:org.pentaho.di.trans.step:BaseStep.Category.Flow",
+    documentationUrl = "i18n:org.pentaho.di.trans.step:BaseStep.DocumentationUrl.MetaInject" )
```
Comment (mdamour1976) [inline plugins/meta-inject-plugin/src/main/java/org/pentaho/di/trans/steps/metainject/MetaInjectMeta.java]: Same for this one:
documentationUrl = "0L0/0Y0/0K0/ETL_Metadata_Injection" )

**Comment by pamval** [comment]
@mdamour1976 pr updated

**Comment by davidmsantos90** [comment]
@mdamour1976 update PR with your sugestions. Now we only need your validation that the link on the MetaInject plugin is working correctly

**Comment by davidmsantos90** [comment]
@mdamour1976 update PR with your sugestions. And with your fix in the pdi-osgi-bridge this should be completed.

**Pull request merged**